app/Jobs/FetchCollectionOpenseaSlug.php

```diff
@@ -37,10 +36,6 @@ public function __construct(
      */
     public function handle(): void
     {
-        Log::info('FetchCollectionOpenseaSlug Job: Processing', [
-            'collection' => $this->collection->address,
-        ]);
```

Comment:
these were added to get some insights into the job (when it runs, what it retrieves). What was the reason for removing them?

Reply:
I'd much rather only leave the "worst case scenario" (no slug has been found). We have logrotate which will remove the logs after some time, and if we log every path that every job can take, then the logs will be rotated even faster, making logging useless.

I've updated the PR to log if OpenSea slug is not found (much more useful to know).